### ISSUE TITLE
False positive with @types/chrome

### ISSUE THREAD

**Issue by WiseBird** [gund/eslint-plugin-deprecation#83]
Usages of the chrome API, defined in this types - https://www.npmjs.com/package/@types/chrome, cause deprecation errors.

E.g. the following code
```ts
chrome.tabs.query({});
```
leads to this error:
```bash
deprecation_chrome/index.ts
  1:1  error  'chrome' is deprecated. Part of the deprecated Chrome Apps platform  deprecation/deprecation
```
whereas the API is not marked as deprecated
```js
/**
 * Gets all tabs that have the specified properties, or all tabs if no properties are specified.
 * @since Chrome 16
 * @return The `query` method provides its result via callback or returned as a `Promise` (MV3 only).
 */
```

Message `Part of the deprecated Chrome Apps platform` is used a few times in the types, but has nothing to do with `chrome.tabs.query`.

Reproduction - https://github.com/WiseBird/deprecation_chrome
